examples/tests/valid/json.w

```diff
@@ -21,6 +21,9 @@ assert(json_mut_obj.get("hello") == message);
 // Assignment from natives
 let some_number: num = 999;
 
+let jj = Json some_number;
+assert(jj == some_number);
```

Comment:
didn't we just say that this shouldn't work?

Reply:
hmm I thought we said this should work 😅. Are you referencing using the variable as the root element? Or the equality check?

Reply:
Can we also add:

```js
let jj1 = Json { foo: some_number };
let jj2 = Json [ some_number, { bar: some_number } ];
```

Reply:
> hmm I thought we said this should work 😅. Are you referencing using the variable as the root element? Or the equality check?

Since `some_number` and `jj` have different types, they should not be equal.

Reply:
Oh yes the equality still needs to be fixed (I planned on separate PR), I just wanted to use this PR to update grammar. To support the identifier as root element in Json.